versioned_docs/version-2.0.0/running-keploy/keploy-templatize.md

```diff
@@ -0,0 +1,37 @@
+---
+id: keploy-templatize
```

Comment:
also you need to add the page on sidebars_2.0.0.json under the appropriate section